.github/workflows/pull-requests.yaml

```diff
@@ -0,0 +1,47 @@
+name: Pull Request
+
+on:
+  [pull_request]
+
+jobs:
+  make-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Determine download URL for latest pack
+        id: pack-download-url
+        uses: actions/github-script@v2
+        with:
+          result-encoding: string
+          script: |
+            return github.repos.getReleaseByTag({
+                owner: "buildpacks",
+                repo: "pack",
+                tag: "v0.20.0"
+            }).then(result => {
+                return result.data.assets
+                  .filter(a => a.name.includes("linux"))
+                  .map(a => a.browser_download_url)[0];
+            })
+
+      - name: Install pack
+        run: |
+          curl -s -L -o pack.tgz ${{ steps.pack-download-url.outputs.result }}
+          tar -xvf pack.tgz
+          
+      - name: Build
+        run: PACK_CMD="$(pwd)/pack" make
+
+      - name: Install func
```

Comment:
This may not be necessary since we're not really concerned with the CLI in this package.

Reply:
Update:  Added a wall of text regarding this in the final comment 😊